stdlib/public/Python/Python.swift

```diff
@@ -39,12 +39,17 @@ typealias OwnedPyObjectPointer = PyObjectPointer
 final class PyReference {
   private var pointer: OwnedPyObjectPointer
 
-  init(owning pointer: OwnedPyObjectPointer) {
+  // This `PyReference`, once deleted, will make no delta change to the
+  // python object's reference count. It will however, retain the reference for
+  // the lifespan of this object.
+  init(_ pointer: OwnedPyObjectPointer) {
     self.pointer = pointer
     Py_IncRef(pointer)
   }
 
-  init(borrowing pointer: PyObjectPointer) {
+  // This `PyReference` adopts the +1 reference and will decrement it in the
+  // future.
```

Comment:
Use same doc comment as corresponding `PythonObject` initializer:
```suggestion
  /// Creates a new instance consuming the specified `PyObject` pointer.
```